Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -95,9 +96,12 @@ extension _RSA.Signing {
         ///
         /// This constructor supports key sizes of 1024 bits or more. Users should validate that key sizes are appropriate
         /// for their use-case.
+        /// Parameters from RSA PSS keys will be stripped.
         /// - Warning: Key sizes less than 2048 are not recommended and should only be used for compatibility reasons.
         public init<Bytes: DataProtocol>(unsafeDERRepresentation derRepresentation: Bytes) throws {
-            self.backing = try BackingPublicKey(derRepresentation: derRepresentation)
+            let sanitizedDer = try SubjectPublicKeyInfo.stripRsaPssParameters(derEncoded: [UInt8](derRepresentation))
```

Comment:
Not a request for changes in this PR, but can you file an issue that asks us to move all our RSA key parsing into Swift ASN1? This is a pragmatic change for now, but as we're _already_ parsing the key we should probably just stick with that now.

Reply:
https://github.com/apple/swift-crypto/issues/269